media-libs/kimageannotator/kimageannotator-0.5.2.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
Please bump to EAPI-8.

Reply:
Thanks for the review.  Done, and retested.